.travis.yml

```diff
@@ -15,8 +15,8 @@ jobs:
   include:
     - stage: "API smoke test"
       name: "QA1"
-      script: node_modules/.bin/newman run tests/API_smoke_test.postman_collection.json -e tests/QA1.postman_environment.json --env-var "loginEmail=$QA1_USER_EMAIL" --env-var "loginPw=$QA1_USER_PASSWORD"
-    - script: node_modules/.bin/newman run tests/API_smoke_test.postman_collection.json -e tests/QA2.postman_environment.json --env-var "loginEmail=$QA2_USER_EMAIL" --env-var "loginPw=$QA2_USER_PASSWORD"
+      script: node_modules/.bin/newman run tests/API_smoke_test.postman_collection.json -e tests/QA1.postman_environment.json --env-var "loginEmail=$QA1_USER_EMAIL" --env-var "loginPw=$QA1_USER_PASSWORD" --env-var "marketoClientId=$MARKETO_CLIENT_ID" --env-var "marketoClientSecret=$MARKETO_CLIENT_SECRET" --env-var "marketoEmailPw=$MARKETO_EMAIL_PASSWORD"
+    - script: node_modules/.bin/newman run tests/API_smoke_test.postman_collection.json -e tests/QA2.postman_environment.json --env-var "loginEmail=$QA2_USER_EMAIL" --env-var "loginPw=$QA2_USER_PASSWORD" --env-var "marketoClientId=$MARKETO_CLIENT_ID" --env-var "marketoClientSecret=$MARKETO_CLIENT_SECRET" --env-var "marketoEmailPw=$MARKETO_EMAIL_PASSWORD"
```

Comment:
so correct me if I am wrong but we only have marketo integration for `Production`? In which case we can ditch the setup for `QA1` and `QA2`